### PR TITLE
kepler release docs

### PR DESCRIPTION
**Pull request by bullenc**
Kelper docs deploy

### CODE REVIEW

**Comment by bullenc** [comment]
need to fix merge conflicts first

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md [hunk 1]
```diff
@@ -36,6 +37,148 @@
...
+    * Saving a cohort that was previously saved now displays the correct message. <!--PEAR-1651-->
+    * Cohorts will now display data in Mutation Frequency, Cohort Builder, and the summary charts even when removing gene/mutation filters from a cohort temporarily results in 0 cases. <!--SV-2414-->
+    * Cohorts now contain the correct cases when created from the cases table by using the "Existing Cohort With Selected Cases" and "Existing Cohort Without Selected Cases" options with a cohort containing gene or mutation filters. <!--PEAR-1043-->
+    * When saving a cohort, the confirmation notification will no longer be automatically dismissed before the saving dialog has closed. <!--SV-2366-->
+* __Cohort Builder__:
+    * Cohort Builder cards for number ranges now display an informative message rather than a spinner when there is no data for the facet. <!--PEAR-1646-->
+    * Removing a custom Cohort Builder card no longer incorrectly removes the associated filters from the current cohort. <!--PEAR-1612-->    
+    * Filters related to numeric values in the Cohort Builder now correctly displays the numbers entered. <!--SV-2383-->
```
Comment (clsackett) [inline docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md:107]: "displays" --> "display"

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md [hunk 2]
```diff
@@ -36,6 +37,148 @@
...
+    * Pass the cohort filter to the lollipop track from the matrix and disco plot label click.
+* __Gene Expression Clustering__:
+    * Enable gene variant legend group filter.
+    * Support creating a single-case cohort.
+    * Supported more clustering and distance calculation methods.
+* __OncoMatrix__:
+    * Fix position errors after OncoMatrix/hierCluster zooming in/out caused by outdated imgBox.
+    * Do not allow hiding all the alteration groups.
+    * Disable the geneset submit button when there there is less than a minNumGenes option (3 for hier cluster, 1 for matrix).
+    * Add to OncoMatrix mutation/cnv buttons all available mutation/cnv classes in all GDC instead of only within current cohort.
```
Comment (clsackett) [inline docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md:135]: "cnv" to "CNV" for consistency

**Review by clsackett — APPROVED** [review]
Small changes, nothing major

**Pull request closed without merging**